src/Compilers/CSharp/Portable/Emitter/Model/ArrayTypeSymbolAdapter.cs

```diff
@@ -43,7 +44,7 @@ IEnumerable<int> Cci.IArrayTypeReference.LowerBounds
 
                 if (lowerBounds.IsDefault)
                 {
-                    return DefaultLowerBounds(this.Rank);
+                    return Enumerable.Repeat(0, Rank);
```

Comment:
There is probably similar code in VB implementation. If we are changing C#, we should change VB as well.